updater/index.php

```diff
@@ -1,4 +1,4 @@
-<?php 
+<?php
```

Comment:
Recommend locating this tool in /modules

Reply:
placed it outside, by considering the functions of this updater @aethelwulffe , need to place it in modules?

Reply:
Everything must be considered.  I have approved, but also playing advocate for alternate points of view.

We all know that the more isolated code is, the easier it is to maintain.  We also know that sometimes things are best deployed in other code.

Example 1. is the calendar.
  It used to be tightly bound within the spaghetti code, even though it was originally an external asset.  Now, it is all in one place, and another tool could be dropped in with a reasonable API interface...and completely remove it.  The more isolated the code, the easier it is to do a maintenance PR and not have it stomped on by someone working on something else.

Reply:
the updater is isolated like calendar :confused: it doesnot interact with any parts of application, i mean it does not function as a component of the application (like calendar). Since it is totally unrelated to codebase, i placed it outside to avoid path conflicts which will occur when file replacement by updater will take place.